task15/index.js

```diff
@@ -0,0 +1,94 @@
+const parent = document.getElementById("js-parent"); 
+const modalPlace = document.getElementById('js-modalPlace');
+
+const renderButtonForParent = () => {
+    const button = document.createElement('input');
+    modalPlace.style.display = "none"; 
+    button.id = "js-openBtn";
+    button.type = "button";
+    button.value = "click";
+    parent.appendChild(button);
+};
+renderButtonForParent();
+
+const openBtn = document.getElementById("js-openBtn");
+openBtn.addEventListener("click", (e) => {
+    modalPlace.className = "modal";
+    modalPlace.style.display = "block";
+    e.target.remove();
+    modalClose();
+});
+
+const modalClose = () => {
+    const closeBtn = document.getElementById("js-closeBtn");
+    closeBtn.addEventListener("click", () => {
+        requestValue();
+    });
+};
+
+const requestValue = () => {
+    const getName = document.getElementById("js-getName").value;
+    const getNumber = document.getElementById("js-getNumber").value;
+
+    if(!getName.match(/\S/g) || getNumber === ""){
+        alert("入力してください");
+    } else {
+        console.log(getName);
+        console.log(getNumber);
+        modalPlace.style.display = "none";
+        init();
+    }
+};
```

Comment:
IMO
The conditional branch "true if false" is a little less readable.
When writing if statements, I think it would be easier to understand if you write the process when it is true.
```suggestion
const requestValue = () => {
  const getName = document.getElementById("js-getName").value;
  const getNumber = document.getElementById("js-getNumber").value;

//もしif文内の条件が全てtureなら処理を進めていく
  if (getName.match(/\S/g) || getNumber) {
    console.log(getName);
    console.log(getNumber);
    modalPlace.style.display = "none";
    init();
    return; //ここで処理を終了
  }
  //そうでなければ入力してくださいを出力
  alert("入力してください");
};
```

Reply:
That's right!
I hadn't thought about readability. I'll be careful next time.
[c9a1e22]